sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -1344,7 +1356,8 @@ def create_deprecated_read(
     consumers  # type: Dict[str, List[operations.Operation]]
 ):
   # type: (...) -> operations.ReadOperation
-  source = iobase.SourceBase.from_runner_api(parameter.source, factory.context)
+  source = iobase.BoundedSource.from_runner_api(
```

Comment:
This shouldn't change the behavior of the code, it merely indicates to the type system that we expect `BoundedSource`, since `SourceBase` is not sufficient here (we need `default_output_coder()`)